themes-default/slim/views/config.mako

```diff
@@ -1,110 +1,12 @@
-            <tr><td><i class="icon16-config-ssl"></i> SSL Version:</td><td>{{sslVersion}}</td></tr>
-            <tr><td><i class="icon16-config-os"></i> OS:</td><td>{{os}}</td></tr>
-            <tr><td><i class="icon16-config-locale"></i> Locale:</td><td>{{locale}}</td></tr>
-            <tr><td>&nbsp;</td><td>&nbsp;</td></tr>
-            <tr class="infoTableSeperator"><td>&nbsp;</td><td>&nbsp;</td></tr>
-            <tr><td><i class="icon16-config-user"></i> User:</td><td>{{localUser}}</td></tr>
-            <tr><td><i class="icon16-config-dir"></i> Program Folder:</td><td>{{programDir}}</td></tr>
-            <tr><td><i class="icon16-config-config"></i> Config File:</td><td>{{configFile}}</td></tr>
-            <tr><td><i class="icon16-config-db"></i> Database File:</td><td>{{dbPath}}</td></tr>
-            <tr><td><i class="icon16-config-cache"></i> Cache Folder:</td><td>{{cacheDir}}</td></tr>
-            <tr><td><i class="icon16-config-log"></i> Log Folder:</td><td>{{logDir}}</td></tr>
-            <tr v-if="appArgs"><td><i class="icon16-config-arguments"></i> Arguments:</td><td><pre>{{prettyPrintJSON(appArgs)}}</pre></td></tr>
-            <tr v-if="webRoot"><td><i class="icon16-config-folder"></i> Web Root:</td><td>{{webRoot}}</td></tr>
-            <tr><td>&nbsp;</td><td>&nbsp;</td></tr>
-            <tr class="infoTableSeperator"><td>&nbsp;</td><td>&nbsp;</td></tr>
-            <tr><td><i class="icon16-config-web"></i> Website:</td><td><app-link :href="githubUrl">{{githubUrl}}</app-link></td></tr>
-            <tr><td><i class="icon16-config-wiki"></i> Wiki:</td><td><app-link :href="wikiUrl">{{wikiUrl}}</app-link></td></tr>
-            <tr><td><i class="icon16-config-github"></i> Source:</td><td><app-link :href="sourceUrl">{{sourceUrl}}</app-link></td></tr>
-            <tr><td><i class="icon16-config-mirc"></i> IRC Chat:</td><td><app-link href="irc://irc.freenode.net/#pymedusa" rel="noreferrer"><i>#pymedusa</i> on <i>irc.freenode.net</i></app-link></td></tr>
-        </table>
-    </div>
-</script>
 <script>
-const component = {
-    name: 'config',
-    metaInfo: {
-        title: 'Help & Info'
-    },
-    template: '#config-route-template',
-    data() {
-        return {
-            appArgs: undefined,
-            branch: undefined,
-            cacheDir: undefined,
-            commitHash: undefined,
-            configFile: undefined,
-            databaseVersion: undefined,
-            dbPath: undefined,
-            githubUrl: undefined,
-            locale: undefined,
-            localUser: undefined,
-            logDir: undefined,
-            os: undefined,
-            programDir: undefined,
-            pythonVersion: undefined,
-            release: undefined,
-            sourceUrl: undefined,
-            sslVersion: undefined,
-            webRoot: undefined,
-            wikiUrl: undefined
-        };
-    },
-    async mounted() {
-        const { data } = await api.get('config/main');
```

Comment:
If you notice this would already hit the config twice since it's got the api check here. I just moved it to a store dispatch instead of a raw api method.